lib/server.js

```diff
@@ -102,9 +102,9 @@ function parsePOST(req) {
 
 function logResponse(opts, response, startTime) {
     const latency = Date.now() - startTime;
-    let logLevel = 'trace/request';
+    let logLevel = 'info/request';
```

Comment:
There is always a trade-off here between more complete information per request but sampled requests, vs. all requests but only the most basic information for each. For debugging abnormal request behavior, in my experience especially request headers and the status code are indispensable. Most response headers are less critical, although occasionally content-type, cache-control and etag have been useful in the past.

One of the most frequent questions tends to be "what kind of request and client is causing this issue", and answering this without some basic headers is not really possible. Sampling on the other hand does not seem to affect the ability to track down high-volume requests much. Single problematic requests tend to cause errors, which are still captured un-sampled (unless a rate limit triggers on error logging).